lib/IDE/CommentConversion.cpp

```diff
@@ -538,17 +544,25 @@ class DoxygenConverter : public MarkupASTVisitor<DoxygenConverter> {
 
 public:
   DoxygenConverter(llvm::raw_ostream &OS, unsigned Indent = 0)
-    : OS(OS), Indent(Indent), IsFreshLine(true) {
+    : OS(OS), Indent(Indent), IsFreshLine(true), IsEmptyComment(true) {
     printIndent();
```

Comment:
I know the initial `Indent` is always `0` in the current usage.
But `DoxygenConverter(OS, 2)` would print something like:
```objc
  ///   Aaa.
///   Bbb.
```
I think, it should be
```objc
  /// Aaa.
  /// Bbb.
```

(or, can we just remove `Indent` from the constructor paramters?)